frontend/src/metabase/common/components/DataPicker/components/TablePicker.tsx

```diff
@@ -138,6 +141,42 @@ export const TablePicker = ({
     [dbId, schemaName, setTableId, onItemSelect, onPathChange],
   );
 
+  const onItemSelectRef = useLatest(onItemSelect);
+  const handleFolderSelectRef = useLatest(handleFolderSelect);
+
+  useEffect(
+    function ensureFolderSelected() {
```

Comment:
the function seems not be called

Reply:
It is, e.g. New > Question > open Tables tab